acceptance/makeMockAgents.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+trap 'kill $(jobs -p)' EXIT
+
+cd ..
+make mockAgent
```

Comment:
I think prefer to rename this script something like startMockAagents.sh and remove this make. Again, the motivation is separation of concerns and I draw an analogy to serviced. Building serviced is separate from starting serviced which is separate from running the test against serviced.   We can instructions for making mockAgent to the README.